server/sonar-server/src/main/java/org/sonar/server/computation/ComputationContext.java

```diff
@@ -50,11 +53,20 @@ public ComponentDto getProject() {
   }
 
   public BatchReport.Metadata getReportMetadata() {
-    Preconditions.checkState(reportMetadata != null, "Report metadata is available after execution of " + ParseReportStep.class);
+    checkState(reportMetadata != null, "Report metadata is available after execution of " + ParseReportStep.class);
```

Comment:
nope, the reportMetadata is set during the parse report step. If we try to access this data before, it throws a exception